load-tests/gradle/src/gatling/scala/Config.scala

```diff
@@ -1,3 +1,4 @@
 object Config {
     val urlUnderTest = "https://ntbs-load-test.e32846b1ddf0432eb63f.northeurope.aksapp.io"
+    val lengthOfTestInMinutes = sys.env.getOrElse("LOAD_TEST_DURATION_IN_MINUTES", "5").toInt
```

Comment:
Default is 5 minutes here because that's better for development. On the github action it's 30 minutes.